Added a section for JSON-encoded results (#1636)
MicrosoftDocs/edge-developer: microsoft-edge/webview2/concepts/distribution.md

If you want to use the Fixed Version distribution mode, you can skip the next couple of sections and navigate to [Details about the Fixed Version runtime distribution mode](#details-about-the-fixed-version-runtime-distribution-mode).

> ## Servicing the WebView2 Runtime through Windows Server Update Services (WSUS)

Reviewer: ```suggestion
### Servicing the WebView2 Runtime through Windows Server Update Services (WSUS)
```
h2, or h3?  check logical nesting

> Refer to the Windows Server Update Services (WSUS) section of the (Managing WebView2 applications)[enterprise.md#windows-server-update-services-wsus] article.

Reviewer: ```suggestion
Refer to the Windows Server Update Services (WSUS) section of the [Managing WebView2 applications](enterprise.md#windows-server-update-services-wsus) article.
```

Reviewer 2: done

> Please refer to the WSUS section of the (Managing WebView2 applications)[https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/enterprise#windows-server-update-services-wsus] article.

Reviewer 2: ```suggestion
Refer to the Windows Server Update Services (WSUS) section of the (Managing WebView2 applications)[enterprise.md#windows-server-update-services-wsus] article.
```

Reviewer 2: I repeated Windows Server Update Services (WSUS) because it is the title of the section.
